frontend_server/lib/server.dart

```diff
@@ -137,7 +136,7 @@ class _FrontendCompiler implements CompilerInterface {
     ArgResults options, {
     IncrementalKernelGenerator generator,
   }) async {
-    _filename = filename;
+    final Uri _filenameUri = Uri.base.resolve(new Uri.file(filename).toString());
```

Comment:
Nit: consider dropping underscore from names of local variables.

Reply:
Good point, done.